include/swift/AST/Expr.h

```diff
@@ -307,12 +307,13 @@ class alignas(8) Expr : public ASTAllocated<Expr> {
     NumCaptures : 32
   );
 
-  SWIFT_INLINE_BITFIELD(ApplyExpr, Expr, 1+1+1+1+1,
+  SWIFT_INLINE_BITFIELD(ApplyExpr, Expr, 1+1+1+1+1+1,
     ThrowsIsSet : 1,
     Throws : 1,
     ImplicitlyAsync : 1,
     ImplicitlyThrows : 1,
-    NoAsync : 1
+    NoAsync : 1,
+    ShouldApplyDistributedThunk : 1
```

Comment:
This needs to be set to `false` in the `ApplyExpr` constructor, or you'll end up with spuriously-distributed calls due to uninitialized memory.

Reply:
Oh man, thanks I totally forgot about that 😱